src/components/Tags/Tag.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { radius, sizes, typography } from '@magnetis/astro-tokens';
+
+import { getTagProperties } from './utils';
+import type { FontWeightType } from '@typings/styles';
+import type { TagType, TagVariant } from './types';
+
+export type TagProps = {
+  variant?: TagVariant;
+  type?: TagType;
+  text: string;
+  testID?: string;
+};
+
+/**
+ * A customizable tag component.
+ *
+ * @param {TagProps} props - The props for the Tag.
+ * @param {TagVariant} props.variant - The variant of the Tag.
+ * @param {TagType} props.type - The type of the Tag.
+ * @param {TagProps} props.text - The text for the Tag.
+ * @param {testID} props.testID - Test ID for testing purposes.
+ *
+ * @returns {JSX.Element} The rendered Tag component.
+ * @example
+ * <Tag type='default' variant='primary' text='tag label' />
+ */
+
+function Tag({ variant = 'neutral', type = 'default', text = 'tag label', testID = 'Tag' }: TagProps) {
+  const { backgroundColor, textColor } = getTagProperties(variant, type);
+
+  const styles = StyleSheet.create({
+    container: {
+      backgroundColor: backgroundColor,
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      minWidth: sizes.great,
+      height: sizes.smaller,
+      borderRadius: radius.mini,
+      paddingVertical: sizes.quark,
+      paddingHorizontal: sizes.nano,
+    },
+    fontStyle: {
+      color: textColor,
```

Comment:
💅 If this prop was called `color`, you could do the following:
```suggestion
      color,
```

Reply:
Fixed in [500aaba](https://github.com/magnetis/astro-native/pull/791/commits/500aabac63e9b2e74ed6332f3c9727f6edc33224)